### ISSUE TITLE
False positive in UnneededCapitalW

### ISSUE THREAD

**Issue by jonas054** [rubocop/rubocop#1263]
In #1229 @mmzoo wrote:

Just FYI: I have a similar issue with the `UnneededCapitalW` cop for:

``` ruby
def dangerous_characters
  %W( \000 \001 \027 \002 \003 \004 \005 \006 \007 \00 \a )
end
```

Leads to a

> Do not use %W unless interpolation is needed.  If not, use %w.
